view/frontend/templates/checkout/onepage/success/paynow_additional_info.phtml

```diff
@@ -16,8 +16,8 @@
     <strong>
         <i>Note:</i>
     </strong> 
-    On your bank app, you will see the payee as <strong>Opn Payments SG Pte. Ltd“</strong>. 
-    Opn Payments is our authorized payment processor to accept PayNow on our behalf.
+    On your bank app, you will see the payee as <strong>Omise Payments SG Pte. Ltd“</strong>. 
```

Comment:
For this one I'm not sure. Let's follow the old one 👍 - https://github.com/omise/omise-magento/pull/395/files#diff-8d39062dbe907e32e104fdf11e44953ec6220d175dd61bb1290d9ab549b8b0b7L19

```suggestion
    On your bank app, you will see the payee as <strong>Omise Payment SG Pte. Ltd“</strong>. 
```